config.yml.default

```diff
@@ -44,6 +44,9 @@ engine:                      # engine settings
 #   log: "./engines/log.txt" # save log of lczero debug output
   silence_stderr: false      # some engines (yes you, leela) are very noisy
 
+http_engine:
+  url: "http://play.lczero.org/getMoveHardcore"
```

Comment:
There's no `#` comment?

Reply:
`http_engine` section is only used if `protocol=http` and I am the site owner ;)